xmtp_id/src/associations/unverified.rs

```diff
@@ -278,6 +279,12 @@ impl UnverifiedSignature {
                 &sig.legacy_key_signature.signature_bytes,
                 sig.signed_public_key_proto.clone(),
             ),
+            UnverifiedSignature::Passkey(sig) => VerifiedSignature::from_passkey(
```

Comment:
This needs to pass in the `signature_text` and use it to verify that the signer was actually signing the right thing.

Reply:
Done